screens/UploadDocument.tsx

```diff
@@ -101,9 +85,12 @@ export default function Dashboard({ navigation, route }: Props) {
         url: `/auth/user`,
         headers: { Authorization: `Bearer ${authData.token}` },
       });
-      const user = response.data.data[0].attributes;
-      setUser(user);
-      setOnBoarded(user?.on_boarded);
+      const user = response.data.data[0];
+      setUser(user.attributes);
+      const upload = Object.values(user.included.verification).every(
+        (val) => val === true
```

Comment:
this is weird, what other value could val be?

Reply:
for new customer's values are false, until the customer uploads documents. The function is a truthy check  if any document is not uploaded it returns false and true only when all documents are true
![image](https://user-images.githubusercontent.com/43110334/165720753-46f9eea2-9d09-44f5-b5c0-68188201dfd8.png)

Reply:
will returning just `val` not do the same thing?